### ISSUE TITLE
When I power off and power on, the phone reboot constantly

### ISSUE THREAD

**Issue by victorfriasv** [Android-L-Porting-Team/Android-L-Mako#28]
After power off and power on the phone, it reboot every 10 seconds on. Not is neccesary introduce pin.

**Comment by Darex2094** [comment]
Is this after a clean install? What build of this rom are you using?

**Comment by bzh2610** [comment]
I have this issue too. Plug your phone into a charger/computer, reboot your phone. When the phone asks for the PIN, unplug your charger, type your code, plug your charger back and it should work.

**Comment by cred05** [comment]
I've experienced the same issue on Beta2 and Beta3, both on clean installs.

**Comment by victorfriasv** [comment]
Yes, I did a clean install. I have this issue in Beta2 and Beta3
The temporary solution is plug the phone when started.

**Comment by Darex2094** [comment]
I can't replicate this bug, both on clean installs, dirty installs with 4.4.3, etc.

**Comment by victorfriasv** [comment]
Anyway I just upload a video of this bug. http://youtu.be/D1O9zzzZW_Q
Also happens in beta3 pre5

**Comment by Sykopompos** [comment]
Known L bug http://m.pocketnow.com/2014/07/08/android-l-as-your-daily-driver

**Comment by benpye** [comment]
Closing as Android L bug.

**Comment by mattjanicki** [comment]
> Known L bug http://m.pocketnow.com/2014/07/08/android-l-as-your-daily-driver

I don't think this is what the L bug refers to.  It seems to be talking about the PIN re-locking the screen instantly, not rebooting.  The author then mentions manually rebooting to fix the PIN re-locking.  Are we sure this is a known L bug?

**Comment by The1andONLYdave** [comment]
I can confirm this bug. 
Clean install from latest stock, after flash it worked yesterday, I set up some apps, rebooted, all working. Now today suddenly bootlooping till lockscreen, I can even unlock it (swype), but it restarts 5 seconds later. Already check2fs'd /data, wiped /cache and dalvik and did even format /system and reinstalled from .zip, then it boots longer, rebuild app (that thing with app xx of yy(somewhat around 90) and bootloop again. Any files I could check what's the cause, like logfiles?
I have no pin/sim card and no unlock code. For me it looks like a service/app starts in background and cause this. Charger connected don't change anything.

**Comment by The1andONLYdave** [comment]
After lot of searching and trying, it is the wifi connection, if I poweroff my router my nexus is working, or if I turn off wifi. I already disabled (with autostart app) every app under android.net.conn.connectivity_change.
It's not a hardware problem, since with clean /data everything works. Will check next week if it happen with different wifi-networks.

**Comment by Sykopompos** [comment]
Clear your known WiFi connection and reenter your info. I swore I seen something mentioned about that causing issues with the auto backup WiFi or tibi backed up ones

**Comment by The1andONLYdave** [comment]
I cleared one wifi, reentered, and after exactly 5 seconds it reboots again. Or do you mean clearing (how?) the whole saved-wifi-database? I too tried blocking _everything_ in afwall+, but no difference.

**Comment by Sykopompos** [comment]
I'm not sure what is going on with it then because this seems to be a very limited issue. Is it a 5ghz router? Have u gotten a logcat so we can try to track it down?

**Comment by Sykopompos** [comment]
Are you guys on 5ghz routers? If so check the second post on the Dev thread and trying changing the WiFi channel

**Comment by The1andONLYdave** [comment]
It's a 2,4 ghz wrt54gl mixed (B and G) wifi with wpa2(tkip) encryption.
The other wifi is a bb10 hotspot.
Logcat is on http://pastebin.com/n5hvnhY2

//edit: My bb10 device allows only 2,4ghz wifi access, so i think it makes only 2,4ghz Hotspots too.

**Comment by Sykopompos** [comment]
this is what is causing your crash right here "android.database.sqlite.SQLiteException: Can't downgrade database from version 589832 to 524296" it looks like an app causing it. I see dropbox and facebook going at the same time

**Comment by The1andONLYdave** [comment]
Got the same error in this logcat http://pastebin.com/34bxAq93
This one is after i boot with a sim-card(without pin) it crashes com.android.phone every 2-3 seconds, but no reboot. 

Dropbox is not installed, do you mean the dropbox folder under /data/system/dropbox? looks like some sort of tombstone- and crashlogs in there. Should i upload them too? But i can't copy it, even with root and open only very few files in it (about 1000 files there).

After uninstalling Facebook, it still reboots 5 sec after connecting.

**Comment by The1andONLYdave** [comment]
Uninstalled every app(except titanium/totalcommander) and disabled google-sync and deactivated all apps, it still reboots ( http://pastebin.com/SieHx3k0 ). 
Any idea where i can find which sqlite database is the one mentioned in logfiles? Then i could try copying it from another install.
//edit: deleted sqlite .db and .db-journal from com.android.providers.telephony and .userdictionary, 2 of 4 got rebuild at next boot, and it's finally working again with both wifi-networks.
Will try to get all apps up again, sync set up, restore twrp and check if deleting this files only, fix it.

**Comment by The1andONLYdave** [comment]
It definetly were the incompatible sqlite databases, probably they came from a carbon restore of userdictionary and sms-mms, but after the restore the messages showed up nowhere so i thought it just failed. why android l uses a older (downgrade) version than 4.4.4 is another question.
Maybe the other Person with crashes after boot/wifi have a similar database problem?

**Comment by mattjanicki** [comment]
So which SQLite databases did you end up deleting?

Edit: Nevermind, I see the update about /data/data/com.android.providers.telephony and .../com.android.providers.userdictionary.  Deleted the files and haven't had a reboot yet, but I suppose we'll see.

**Comment by The1andONLYdave** [comment]
Folder /data/data/com.android.providers.telephony/databases/This 6: mmssms.db mmssms.db-journal telephony.db telephony.db-journal blacklist.db blacklist.db-journal (blacklist could be something similar, don't get rebuild, so can't check atm)And folder /data/data/com.android.providers.userdictionary/databases/ all two files (.db and .db-journal can't check exactly name too)                                                                                                                                                                                                                                        Von: SurlyDuffGesendet: Samstag, 12. Juli 2014 15:07An: Android-L-Porting-Team/Android-L-MakoAntwort an: Android-L-Porting-Team/Android-L-MakoCc: David-Lee KulschBetreff: Re: [Android-L-Mako] When I power off and power on, the phone reboot constantly (#28)So which SQLite databases did you end up deleting?

—Reply to this email directly or view it on GitHub.

**Comment by dnewsholme** [comment]
Hi guys.

Lightflow notification control is the cause of this.

http://reactle.blogspot.co.uk/2014/07/l-preview-details.html?m=0

L Preview Details
Version 3.20.107 of Light Flow adds in compatibility for the Android L preview.

At the moment there's a few issues, but versions prior to this cause a boot loop in android (ie shortly after the phone boot's it'll reboot with lightflow installed)

This seems to be an issue with L I've raised an issue on the android L issue tracker here: https://code.google.com/p/android-developer-preview/issues/detail?id=667

**Comment by The1andONLYdave** [comment]
I don't have/had Lightflow or similar installed under Android L, only the clear-notification apk from clockworkmod team.

**Comment by dnewsholme** [comment]
Does the clear all notifications display an icon in the top left always? Even when notification tray is closed?

If it doesn't it sounds like it could cause the issue.

**Comment by Sykopompos** [comment]
so it was the sqlite databases from a backup that was causing this?

**Comment by mattjanicki** [comment]
Seems so.  Haven't had any other issues recently.

**Comment by Sykopompos** [comment]
Well now we know that we got a sqlite incompatibly between 4.4.4 and L now though

**Comment by The1andONLYdave** [comment]
Maybe you can put it at the xda-thread? That restoring SMS/MMS can cause bootloops at the moment.
